webpack.dev.js

```diff
@@ -10,7 +10,7 @@ module.exports = merge(common, {
     devServer: {
         contentBase: APP_DIR,
         // writeToDisk: true,
-        // host: '192.168.0.180',
+        host: '0.0.0.0',
```

Comment:
Had to change the host to 0.0.0.0 to make this public. Is there a way to do this when calling webpack-dev-server in package.json?

Reply:
good call - and there is! (`webpack-dev-server --host XXXX` etc). Interesting issue/thread about its finer points [here](https://github.com/webpack/webpack-dev-server/issues/882). We should test this further, I understand Jared had a mixed experience but remote access to the local serve would be a big win.